### PR TITLE
REM-396

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
sa-price-groups

### CODE REVIEW

File: src/pages/sa-price-groups/forms/PriceGroupsForm.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  }, [])
+
+  return (
+    <FormShell resourceId={ResourceIds.PriceGroups} form={formik} maxAccess={maxAccess} editMode={editMode}>
+      <VertLayout>
+        <Grow>
+          <Grid container spacing={4}>
+            <Grid item xs={12}>
+              <CustomTextField
+                name='reference'
```
Comment (RebeccaFrem02) [inline src/pages/sa-price-groups/forms/PriceGroupsForm.js:107]: the reference is required

File: src/pages/sa-price-groups/forms/PriceGroupsForm.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                onChange={formik.handleChange}
+                onClear={() => formik.setFieldValue('minUnitPrice', '')}
+                error={formik.touched.minUnitPrice && Boolean(formik.errors.minUnitPrice)}
+                allowNegative={false}
+                decimalScale={3}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <CustomNumberField
+                name='maxUnitPrice'
```
Comment (RebeccaFrem02) [inline src/pages/sa-price-groups/forms/PriceGroupsForm.js:145]: check if you need to add require here

File: src/pages/sa-price-groups/forms/PriceGroupsForm.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    initialValues: {
+      recordId: null,
+      reference: '',
+      name: '',
+      minUnitPrice: '',
+      maxUnitPrice: ''
+    },
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
```
Comment (RebeccaFrem02) [inline src/pages/sa-price-groups/forms/PriceGroupsForm.js:35]: check the validation 
Screenshot 2024-07-31 110431

File: src/pages/sa-price-groups/forms/PriceGroupsForm.js (hunk 2 above)
Comment (Alinadimmokdad) [reply src/pages/sa-price-groups/forms/PriceGroupsForm.js:145]: no need , they are not required

File: src/pages/sa-price-groups/forms/PriceGroupsForm.js (hunk 3 above)
Comment (Alinadimmokdad) [reply src/pages/sa-price-groups/forms/PriceGroupsForm.js:35]: done

**Pull request merged**